### PR TITLE
Adds Learn CSS color module

### PR DESCRIPTION
**Pull request by rachelandrew**
Fixes: #4782

Adds an initial draft of the color module.

### CODE REVIEW

**Review by argyleink — APPROVED** [review]
pending the todo's are completed, this reads really well 👍🏻 
demo's look stellar.
how can i help with the self assessment?

**Review by una — COMMENTED** [review]
Sweet! This one looks great. I just made a few comments regarding whether or not to add commas. I'm in the comma club, but I could be convinced.

**Comment by rachelandrew** [comment]
> Sweet! This one looks great. I just made a few comments regarding whether or not to add commas. I'm in the comma club, but I could be convinced.

There was discussion about this in the doc IIRC and the decision was made not to use the commas there. @andy-piccalilli @argyleink what do you think?

**Comment by andy-piccalilli** [comment]
I don't really have a strong opinion on commas to be honest. I just want the article to be as clear as possible so with that in mind, I'll say lets commit those tweaks @una made.

**Comment by andy-piccalilli** [comment]
Ready for a last pass and self assessments 👍

**Review by andy-piccalilli — APPROVED** [review]

**Comment by argyleink** [comment]
while we each may have strong or light feelings about commas, the CSSWG does have a strong (unified) opinion, we should follow and not mix opinions?

**Comment by una** [comment]
@argyleink — the vast majority usage is commas currently (in production, in tutorials, etc.). They were introduced first and have the most browser support. For an intro course, I think its okay to show commas by default and have the note (which is included here) that you can also use spaces now.

**Comment by andy-piccalilli** [comment]
Another point that's just occurred to me is that the vast majority of tutorials, samples and StackOverflow answers will probably still use commas too, so for a beginner, we could accidentally create a barrier for them by using the new syntax.

File: src/site/content/en/learn/css/color/index.md [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+```
+
+RGB colors are defined with the
+[`rgb()`](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/rgb()) color function,
+using either numbers or percentages as parameters.
+The numbers need to be within the **0-255** range and the percentages are between **0% and 100%‌**.
+RGB works on the 0-255 scale,
+so 255 would be equivalent to 100%, and 0 to 0%.
+
+To set black in RGB, define it as `rgb(0 0 0)`,
```
[the remark this answers is not in the document]
Comment (una) [reply src/site/content/en/learn/css/color/index.md:114]: last one?

**Review by una — APPROVED** [review]

**Pull request merged**